app/src/main/java/org/simple/clinic/patient/recent/RecentPatientsView.kt

```diff
@@ -17,6 +17,8 @@ import io.reactivex.subjects.PublishSubject
 import kotterknife.bindView
 import org.simple.clinic.R
 import org.simple.clinic.activity.TheActivity
+import org.simple.clinic.recentpatient.RecentPatientsScreenKey
```

Comment:
The `patient` package is meant for placing classes related to the business logic/data only. Can you move the View related classes into a separate package for now?

For what it's worth, I feel that the way we organise our packages could be more obvious. Let's discuss this at some point.

Reply:
done